app/components/Comments/__tests__/CommentTree.spec.js

```diff
@@ -6,6 +6,9 @@ import comments from './fixtures/comments';
 import { generateTreeStructure } from '../../../utils';
 
 describe('<CommentTree />', () => {
+  beforeAll(() => {
+    window.getSelection = () => {};
```

Comment:
Make a comment here so folks from the "gemene hop" can understand what this is.

Also, does it make sense to have it here?